app/views/categories/_form.html.erb

```diff
@@ -46,8 +46,25 @@
     </div>
   <% end %>
 
+  <div class="form-group">
+    <%= form.label :tag_list, "Tags" %>
```

Comment:
Yes, but we still don't know all the purposes for which it will be used. Initially, the (ancient) CMS will use it for filtering in the dropdowns, but that's just one possible application.